README.md

```diff
@@ -109,8 +109,34 @@ export ETH_SIGNING_KEY=<Your ETH SIGNING KEY>
 pigeon start
 ```
 
-- Remember to run pigeon as a systemd Service! If you have a good systemd service implementation for Ubuntu, please make a PR on this README and we will add it.
-
+#### Using systemd service
+Create service file
+
+```shell script
+tee <<EOF >/dev/null /etc/systemd/system/pigeond.service
+[Unit]
+Description=Pigeon daemon
+After=network-online.target
+
+[Service]
+Environment="PALOMA_KEYRING_PASS=$PALOMA_KEYRING_PASS"
```

Comment:
Does this assume that the person running this command has $PALOMA_KEYRING_PASS and other env variables set when running this command? (@aditya-manit )

Reply:
Yes, we are setting it here before running that command. Line 103 to line 110